File: print-manager-dotnet-core-cs/DisposeExample.cs
using ceTe.DynamicPDF.Printing;

namespace print_manager_dotnet_core_cs
{
    internal class DisposeExample
    {
        public static void Run()
        {
            PrintJob printJob = new PrintJob(Printer.Default, Util.GetPath("Resources/DocumentA.pdf"));
            printJob.Print();
            printJob.Dispose();

            InputPdf pdf = new InputPdf(Util.GetPath("Resources/DocumentA.pdf"));
            printJob = new PrintJob(Printer.Default, pdf);
            printJob.Print();
            pdf.Dispose();
        }
    }
}

File: print-manager-dotnet-core-vb/DisposeExample.vb
Imports ceTe.DynamicPDF.Printing

Friend Class DisposeExample

    Public Shared Sub Run()

        Dim printJob As New PrintJob(
                Printer.Default,
                Util.GetPath("Resources/DocumentA.pdf"))

        printJob.Print()
        printJob.Dispose()

        Dim pdf As New InputPdf(
                Util.GetPath("Resources/DocumentA.pdf"))

        printJob = New PrintJob(Printer.Default, pdf)

        printJob.Print()

        pdf.Dispose()

    End Sub

End Class
